src/main/java/gov/nasa/jpl/mbee/mdk/mms/validation/ElementValidator.java

```diff
@@ -100,7 +98,7 @@ public void run(ProgressStatus progressStatus) {
             serverElements = new LinkedList<>();
             serverElementMap = new HashMap<>();
         } else {
-            serverElementMap = serverElements.stream().filter(json -> json.has(MDKConstants.ID_KEY) && json.get(MDKConstants.ID_KEY).isTextual()).collect(Collectors.toMap(json -> json.get(MDKConstants.ID_KEY).asText(), Function.identity()));
+            serverElementMap = serverElements.stream().filter(json -> json.has(MDKConstants.ID_KEY) && json.get(MDKConstants.ID_KEY).isTextual() && !json.get(MDKConstants.ID_KEY).asText().startsWith(MDKConstants.HIDDEN_ID_PREFIX)).collect(Collectors.toMap(json -> json.get(MDKConstants.ID_KEY).asText(), Function.identity()));
```

Comment:
Should be separated into its own `filter` pipeline step. If you're feeling up for some continuous improvement, the two usages of `removeServerObjectNodeUsingIdPrefix` can be replaced with this approach to avoid unnecessary iterations.

Reply:
Based on further investigation and discussion at the dev tag-up, `hidden_` elements are only now popping up because of a local experiment to try using the server's `getAllElements` instead of a recursive search for performance reasons. This causes `hidden_` elements in the `View Instances Bin` and the `Holding Bin` to be returned while the recursive search method would/does not. Since the resolution to the performance issues is being addressed server-side, the recursive search can stick around and this change should no longer be needed.